src/node.rs

```diff
@@ -301,36 +328,404 @@ impl Taffy {
+
+        let node = taffy.new_with_children(FlexboxLayout::default(), &[]).unwrap();
+        assert_eq!(taffy.style(node).unwrap().display, Display::Flex);
+
+        taffy.set_style(node, FlexboxLayout { display: Display::None, ..FlexboxLayout::default() }).unwrap();
+        assert_eq!(taffy.style(node).unwrap().display, Display::None);
+    }
+    #[test]
+    fn test_style() {
+        let mut taffy = Taffy::new();
+
+        let style =
+            FlexboxLayout { display: Display::None, flex_direction: FlexDirection::RowReverse, ..Default::default() };
+
+        let node = taffy.new_with_children(style, &[]).unwrap();
+
+        let res = taffy.style(node);
+        assert!(res.is_ok());
+        assert!(res.unwrap() == &style);
+    }
+    #[test]
+    fn test_layout() {
+        let mut taffy = Taffy::new();
+        let node = taffy.new_with_children(FlexboxLayout::default(), &[]).unwrap();
+
+        // TODO: Improve this test?
+        let res = taffy.layout(node);
+        assert!(res.is_ok());
+    }
+
+    #[test]
+    fn test_mark_dirty() {
+        let mut taffy = Taffy::new();
+        let child0 = taffy.new_with_children(FlexboxLayout::default(), &[]).unwrap();
+        let child1 = taffy.new_with_children(FlexboxLayout::default(), &[]).unwrap();
+        let node = taffy.new_with_children(FlexboxLayout::default(), &[child0, child1]).unwrap();
+
+        taffy.compute_layout(node, Size::undefined()).unwrap();
+
+        assert_eq!(taffy.dirty(child0).unwrap(), false);
+        assert_eq!(taffy.dirty(child1).unwrap(), false);
+        assert_eq!(taffy.dirty(node).unwrap(), false);
+
+        taffy.mark_dirty(node).unwrap();
+        assert_eq!(taffy.dirty(child0).unwrap(), false);
+        assert_eq!(taffy.dirty(child1).unwrap(), false);
+        assert_eq!(taffy.dirty(node).unwrap(), true);
+
+        taffy.compute_layout(node, Size::undefined()).unwrap();
+        taffy.mark_dirty(child0).unwrap();
+        assert_eq!(taffy.dirty(child0).unwrap(), true);
+        assert_eq!(taffy.dirty(child1).unwrap(), false);
+        assert_eq!(taffy.dirty(node).unwrap(), true);
+    }
+
+    #[test]
+    fn compute_layout_should_produce_valid_result() {
+        let mut taffy = Taffy::new();
+        let node_result = taffy.new_with_children(
+            FlexboxLayout {
+                size: Size { width: Dimension::Points(10f32), height: Dimension::Points(10f32) },
+                ..Default::default()
+            },
+            &[],
+        );
+        assert!(node_result.is_ok());
+        let node = node_result.unwrap();
+        let layout_result = taffy.compute_layout(node, Size { width: Some(100.), height: Some(100.) });
+        assert!(layout_result.is_ok());
+    }
+
+    // #[test]
+    // fn compute_layout_should_produce_error_on_invalid() {
+    //     // ensure taffy.compute_layout will produce InvalidNodeError if operating on an invalid node
+    //     // todo!("Test is missing");
+    // }
```

Comment:
Is this still needed?

Reply:
I'll remove it. The only path we are not covering atm is that anything that can produce an error is actually doing so. 

I think we can add those later once we have more tests overall and better tooling for seeing what coverage looks like.